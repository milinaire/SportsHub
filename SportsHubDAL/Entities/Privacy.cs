using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace SportsHubDAL.Entities
{
    public class Privacy
    {
        [Key]
        public int LanguageId { get; set; }
        public Language Language { get; set; }
        public string Headline { get; set; }

        public string Text { get; set; }
    }
}
