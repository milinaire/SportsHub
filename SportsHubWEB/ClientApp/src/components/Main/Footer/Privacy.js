import React, { Component } from "react";
import "./Footer.css";

export class Privacy extends Component {
  state = {
    Headline: "Privacy",
    Text:
      "Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, ",
  };
  render() {
    return (
      <div>
        <h1>{this.state.Headline}</h1>
        <p>{this.state.Text}</p>
      </div>
    );
  }
}
