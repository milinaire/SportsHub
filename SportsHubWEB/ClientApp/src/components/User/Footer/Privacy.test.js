
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import {Privacy} from "./Privacy";



describe('Main\Footer\Privacy test', () => {
  it('Main\Footer\Privacy should match snapshot', () => {
    const component = renderer.create(<Privacy/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
