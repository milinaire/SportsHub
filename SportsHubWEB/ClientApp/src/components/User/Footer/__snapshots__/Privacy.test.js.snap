// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooterPrivacy test MainFooterPrivacy should match snapshot 1`] = `
<div
  className="main-part-footer"
>
  <h1>
    Privacy
  </h1>
  <p>
    Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, 
  </p>
</div>
`;
